opslevel/validators.go

```diff
@@ -14,17 +14,14 @@ import (
 // OpsLevel ID String Validator
 type idStringValidator struct{}
 
-// Description describes the validation in plain text formatting.
```

Comment:
Removed these comments since they are not helpful and made the code harder to read.